.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           - macOS-latest
         arch:
           - x64
+          - aarch64
```

Comment:
This doesn't make sense without excluding the impossible combinations

Reply:
Yes, but I don't know enough about GHA on how to do that :)

Reply:
#32